docs/applications/deluge.md

```diff
@@ -0,0 +1,60 @@
+# Deluge
+
+[Deluge](http://deluge-torrent.org/) is a lightweight, Free Software, cross-platform BitTorrent client.
+
+* Full Encryption
+* WebUI
+* Plugin System
+* Much more...
+
+
+[![deluge](https://avatars2.githubusercontent.com/u/6733935?v=3&s=200)](http://deluge-torrent.org/)
+
+
+## Usage
+
+Set `deluge_enabled: true` in your `roles/deluge/main.yml` file.
```

Comment:
Needs to be set in `inventories/[your_inventory]/group_vars/nas.yml`, role/task files shouldn't be edited.